src/fragments/lib/react-native-polyfills.mdx

```diff
@@ -1,19 +0,0 @@
-### Install Amplify and its dependencies
```

Comment:
This is only used by Interactions and can be safely removed, moved to `src/fragments/lib/interactions/react-native/frontend.mdx` to add interactions specific install instructions